src/components/ContentPreview/ContentPreview.js

```diff
@@ -229,6 +229,8 @@ class ContentPreview extends PureComponent<Props, State> {
         }
         // Don't destroy the cache while unmounting
         this.api.destroy(false);
+
+        document.removeEventListener('keydown', this.onKeyDown);
```

Comment:
Can you add this only when preview is fullscreen? Do you need to add it to document or can you add to the preview container's dom node

Reply:
Agreed. Can you add it to `measureRef`, maybe?